desk.py

```diff
@@ -0,0 +1,88 @@
+import threading
+
+'''
+This class will essentially be a model for a desk, containing desk name, id, etc.
+Note: pin information will be stored in desk_map
+'''
+
+
+class Desk:
+    __id = -1
+    __desk_name = ""
+    __occupied = False
+    __desk_observers = None
+
+    # Use this for synchronizing the input_received function, since pin interrupt callbacks are async,
+    # we get some weird behaviour if we dont synchronize it
+    __lock = threading.Lock()
+
+    '''
+    Initialize Desk Object
+    '''
+    def __init__(self, desk_name, id=-1, occupied=False):
+        self.__desk_name = desk_name
+        self.__occupied = occupied
+        self.__id = id
+        self.__desk_observers = set()
+
+    '''
+    Print Desk Info  *such wow*
+    '''
+    def print_info(self):
+        print "Desk Name:", self.__desk_name, ", Currently Occupied:", self.__occupied
```

Comment:
Wouldn't this produce an extra space after the `desk_name` and before the comma?